.gitignore

```
@@ -0,0 +1 @@
+.idea/*
```

`.idea/` implies all items under that directory
```suggestion
*.idea
.idea/
```

---

That's what I want :)